collections/nemo_asr/nemo_asr/audio_preprocessing.py

```diff
@@ -24,7 +24,10 @@
 from abc import abstractmethod
 import math
 import torch
-import torchaudio
+try:
+    import torchaudio
+except ModuleNotFoundError:
+    print('Could not import torchaudio. Some features might not work.')
```

Comment:
Can we set a global var here and make AudioToSpectrogramPreprocessor and AudioToMFCCPreprocessor raise an error if this global var is set?

Reply:
fixed